### PR TITLE
Fix false positive in `operator_usage_whitespace`

### LINKED ISSUE #1028: 'operator_usage_whitespace' rule conflicts with Image Literal
When using image literal, It's should not triggered

### PR DESCRIPTION
**Pull request by marcelofabri**
Fixes #1028

### CODE REVIEW

File: Source/SwiftLintFramework/Rules/OperatorUsageWhitespaceRule.swift [hunk 1]
```diff
@@ -105,7 +107,7 @@ public struct OperatorUsageWhitespaceRule: OptInRule, CorrectableRule, Configura
             zeroSpaces + trailingVariableOrNumber
         let excludingPattern = "(?:\(genericPattern)|\(validRangePattern))"
 
-        let kinds = SyntaxKind.commentAndStringKinds()
+        let kinds = SyntaxKind.commentAndStringKinds() + [.objectLiteral]
```
Comment (jpsim) [inline Source/SwiftLintFramework/Rules/OperatorUsageWhitespaceRule.swift]: I would suspect that other rules that exclude comments and strings would also need to exclude object literals.

This is why checking for explicit matching kinds is always preferable to excluding.

Comment (marcelofabri) [inline Source/SwiftLintFramework/Rules/OperatorUsageWhitespaceRule.swift]: I agree, but I'm not sure what we should do. Maybe rename `commentAndStringKinds` and make it return `.objectLiteral` as well?

Comment (jpsim) [inline Source/SwiftLintFramework/Rules/OperatorUsageWhitespaceRule.swift]: yes, basically "non-code kinds", and check to see if there are even more kinds that fall into that category.

Comment (jpsim) [inline Source/SwiftLintFramework/Rules/OperatorUsageWhitespaceRule.swift]: I've looked at all the other rules using `commentAndStringKinds()` and I don't think any of them would benefit from also including image literals. So let's keep it just here for now.

**Pull request merged**